README.md

```diff
@@ -24,42 +31,46 @@ This version of the library is compatible with .NET Framework version 4.5 and ab
 
 2. Import the package (note that the namespace is different from the package name):
 
-        using LaunchDarkly.Client.Redis;
+        using LaunchDarkly.Sdk.Server.Integrations;
 
```

Comment:
In the instructions below, I've followed the format and wording that we used in the equivalent Java integrations, with adjustments for C# syntax etc.